pages/1.11/release-notes/1.11.0-rc1/index.md

```diff
@@ -86,14 +86,14 @@ Please try out the new features and updated data services. Provide any feedback
 - Non-essential tasks now supported for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS. If non-essential tasks fail, the entire node will not fail.
 
 # <a name="known-issues"></a>Known Issues and Limitations
```

Comment:
We need to include another `Known Issue`.
`systemd-networkd` must be `enabled` for the dcos networking to be working CORE-OS. (https://jira.mesosphere.com/browse/DCOS_OSS-2130) 

/cc: @mellenburg